### ISSUE TITLE
Pagination gets messed up after opening a new page from link on list view

### ISSUE THREAD

**Issue by lmejiasd** [mendixlabs/list-view-controls#33]
Steps to reproduce the issue:

1. Create a list view control and add a **pagination** widget.
2. Create a new a page with a close page button. **Don’t use modal layout for this new page.**
3. Set the list view **page size** to 5.
4. Add a column in list view (link) that opens the new page created in step 2.
5. Run app and make sure list view holds more than 5 items.
6. Go to **last page**. (By using the pagination widget)
7. Click on the link column to open the new page. 
8. Close the page by using close page button.
9. The expected behavior should be to see 5 records on the first page, however, you will see **all** records instead.  For some reason list view page size gets ignored. Also, you will see the "Load more.." button on list view displayed.

**Testing Environment:**

- Mendix Modeler 7.18.1
- Firefox
- Chrome
- IE

**Comment by djkwagala** [comment]
Hi Luis,
Thanks for reporting this issue. we have a [pre-release candidate](https://github.com/FlockOfBirds/list-view-controls/releases/download/v1.3.0-beta/ListViewControls.mpk) which should solve this issue. Kindly try it out.

Regards
Derrick

**Comment by lmejiasd** [comment]
I just installed the pre-release candidate, but I'm getting the following error while replicating scenario depicted above:

```
t is not a function _renderData/</<@http://localhost:8080/mxclientsystem/mxui/mxui.js?636791040871426974:51:177958
o@http://localhost:8080/mxclientsystem/mxui/mxui.js?636791040871426974:20:27660
t.exports.collect@http://localhost:8080/mxclientsystem/mxui/mxui.js?636791040871426974:20:27843
collect@http://localhost:8080/mxclientsystem/mxui/mxui.js?636791040871426974:31:15848
_loadListItems@http://localhost:8080/mxclientsystem/mxui/mxui.js?636791040871426974:51:178841
_renderData/<@http://localhost:8080/mxclientsystem/mxui/mxui.js?636791040871426974:51:177603
o@http://localhost:8080/mxclientsystem/mxui/mxui.js?636791040871426974:20:27660
t.exports.collect@http://localhost:8080/mxclientsystem/mxui/mxui.js?636791040871426974:20:27843
t.exports.ready@http://localhost:8080/mxclientsystem/mxui/mxui.js?636791040871426974:31:60187
_renderData@http://localhost:8080/mxclientsystem/mxui/mxui.js?636791040871426974:51:177530
advice@http://localhost:8080/mxclientsystem/mxui/mxui.js?636791040871426974:5:54174
a@http://localhost:8080/mxclientsystem/mxui/mxui.js?636791040871426974:5:53952
r</b</t.prototype.findListView@http://localhost:8080/widgets/com/mendix/widget/custom/listviewcontrols/Pagination.js?636791040871426974:12:8060
r@http://localhost:8080/mxclientsystem/mxui/mxui.js?636791040871426974:20:28966
```

**Comment by Andries-Smit** [comment]
Dear Luis,

We completed the persisting state and released it as v1.3.0 in the app store.
https://appstore.home.mendix.com/link/app/105694/

Please let us know if is solve you error too,
Cheers, 
Andries
